pygtracker/pygtracker.py

```diff
@@ -23,6 +24,23 @@ def __init__(self):
     # End: Summarize the grades by courses
 
     # Start: Summarise the grades by students
+    def student_ranker(course_id="all", n=10, descending=True):
+        """[Calculates the average grade for a specified number of students and ranks them in descending order for a specific course or for the entire program completed thus far]
+
+        Args:
```

Comment:
I think we are following numpy style? You can refer to the docstring example in individual assignment one for the format to follow